src/scraper.py

```diff
@@ -0,0 +1,47 @@
+import feedparser
+
+AUTHORIZED_TOPICS = [
+    "WORLD",
+    "NATION",
+    "BUSINESS",
+    "TECHNOLOGY",
+    "ENTERTAINMENT",
+    "SPORTS",
+    "SCIENCE",
+]
```

Comment:
is there a link to a reference to all valid topics from google news ?

Reply:
Here is the only info I found: https://stackoverflow.com/questions/51537063/url-format-for-google-news-rss-feed

Reply:
I added a comment with the link.